app/assets/javascripts/modules/moj.matchVisitors.js

```diff
@@ -0,0 +1,206 @@
+(function() {
+  'use strict';
+
+  moj.Modules.MatchVisitors = {
+
+    init: function() {
+      this.cacheEls();
+      this.bindEvents();
+    },
+
+    cacheEls: function() {
+      this.notContactCheckbox = 'input[id*="not_on_list"]';
+      this.bannedCheckbox = 'input[type="checkbox"][id*="banned"]';
+
+      this.$el = $('.js-visitorList.nomis-enabled');
+      this.$noAdultMessage = this.$el.find('.js-noAdults');
+      this.$notAllMessage = this.$el.find('.js-notAllProcessed');
+      this.totalVisitors = this.$el.find('select').length;
+    },
+
+    bindEvents: function() {
+      this.$el.find('select').on('change', $.proxy(this.changeSelect, this));
+      this.$el.find(this.notContactCheckbox).on('change', $.proxy(this.changeNotOnList, this));
+      this.$el.find(this.bannedCheckbox).on('change', $.proxy(this.changeBanned, this));
+    },
+
+    changeSelect: function(e) {
+      var el = e.currentTarget,
+        contactData = $(el).find(':selected').data('contact'),
+        adding = el.value == '0' ? true : false,
```

Comment:
Is this ternary expression necessary? doesn't `el.value == '0'` return true or false?